pylint/checkers/spelling.py

```diff
@@ -304,7 +306,7 @@ def open(self):
             self.spelling_dict = enchant.Dict(dict_name)
 
         if self.linter.config.spelling_store_unknown_words:
-            self.unknown_words = set()
+            self.unknown_words: set = set()
```

Comment:
Do you know the type of the things in the `set`?

Reply:
Yes it's strings.
```suggestion
            self.unknown_words: set[str] = set()
```